mne/io/eeglab/eeglab.py

```diff
@@ -131,6 +134,20 @@ def read_raw_eeglab(input_fname, montage=None, preload=False, eog=(),
         Names or indices of channels that should be designated
         EOG channels. If 'auto', the channel names containing
         ``EOG`` or ``EYE`` are used. Defaults to empty tuple.
+    event_id : dict | None
+        The ids of the events to consider. If None (default),
+        `event_id_func` (see below) is called on every event value. If dict,
+        the keys will be mapped to trigger values on the stimulus channel
+        Keys are case-sensitive.
+        Example: {'SyncStatus': 1; 'Pulse Artifact': 3}.
+    event_id_func : callable | str | None
```

Comment:
 Depends on what exactly you mean. As an example, the current eeglab example data file has two trigger types: 'rt' and 'square'. Others have a bunch of brainvision - like triggers (s101, s102 etc). We need to accommodate for all of these. 